{{cookiecutter.project_slug}}/apps/pages/models.py

```diff
@@ -10,8 +13,61 @@ class StandardPage(Page):
 
     content_panels = Page.content_panels + [StreamFieldPanel("content")]
 
+    share_description = models.TextField(
+        blank=True,
+        max_length=150,
+        help_text="""
+        Description/preview shown when this page is shared on social media.
+        """,
+    )
+    share_image = models.ForeignKey(
+        "wagtailimages.Image",
+        blank=True,
+        null=True,
+        on_delete=models.PROTECT,
+        related_name="+",
+        help_text="""
+        Image shown when this page is shared on social media.
+        """,
+    )
+
+    # Promote panels
+    promote_panels = Page.promote_panels + [
+        MultiFieldPanel(
+            [FieldPanel("share_description"), ImageChooserPanel("share_image")],
```

Comment:
Can we add a trailing comma to the list, might be easier to read quickly

Reply:
would that be `FieldPanel("share_description"), ImageChooserPanel("share_image"),],`?